src/components/FilterBadges/index.tsx

```diff
@@ -11,9 +11,13 @@ const { parseQueryString } = queryString;
 
 interface FilterBadgesProps {
   queryParamsToDisplay: Set<string>;
+  onRemoveAnalytics?: () => void;
+  onClearAllAnalytics?: () => void;
```

Comment:
nit. `onRemove` and `onClearAll`. I think theres value in exposing these callbacks and theres no need to tie them to analytics